Hello World example doesn't build

**Describe the bug**
The hello world example doesn't build.

**To Reproduce**
copied and pasted the hello world example into a new project, then tried to build it.

**Expected behavior**
... I expected it to build.

**Screenshots or pasted code**
matt:~/code/my_first_ggez_app# cargo run
   Compiling my_first_ggez_app v0.1.0 (/home/matt/code/my_first_ggez_app)
error[E0277]: the trait bound `mint::vector::Point2<f32>: std::convert::From<cgmath::point::Point2<f32>>` is not satisfied
  --> src/main.rs:46:9
   |
46 |         graphics::draw(ctx, &self.text, (dest_point,))?;
   |         ^^^^^^^^^^^^^^ the trait `std::convert::From<cgmath::point::Point2<f32>>` is not implemented for `mint::vector::Point2<f32>`
   |
   = help: the following implementations were found:
             <mint::vector::Point2<T> as std::convert::From<[T; 2]>>
             <mint::vector::Point2<T> as std::convert::From<mint::vector::Vector2<T>>>
   = note: required because of the requirements on the impl of `std::convert::Into<mint::vector::Point2<f32>>` for `cgmath::point::Point2<f32>`
   = note: required because of the requirements on the impl of `std::convert::From<(cgmath::point::Point2<f32>,)>` for `ggez::graphics::drawparam::DrawParam`
   = note: required because of the requirements on the impl of `std::convert::Into<ggez::graphics::drawparam::DrawParam>` for `(cgmath::point::Point2<f32>,)`
   = note: required by `ggez::graphics::draw`

error: aborting due to previous error

For more information about this error, try `rustc --explain E0277`.
error: Could not compile `my_first_ggez_app`.

To learn more, run the command again with --verbose.
matt:~/code/my_first_ggez_app#

**Hardware and Software:**
This is with Stable 1.37 running in Linux

--- a/docs/BuildingForEveryPlatform.md
+++ b/docs/BuildingForEveryPlatform.md
@@ -32,13 +32,14 @@ put it in a subdirectory called `resources` in your project root:
 ```sh
 mkdir resources
 cd resources
-wget https://raw.githubusercontent.com/ggez/ggez/master/resources/DejaVuSerif.ttf
+wget https://raw.githubusercontent.com/ggez/ggez/master/resources/LiberationMono-Regular.ttf
 ```
 
 Then edit your `Cargo.toml` with your favorite super duper editor and under `[dependencies]` add:
 
 ```
 ggez = "0.5"
+glam = { version = "0.8.6", features = ["mint"] }
 ```
 
 Now run `cargo run` and it should build

--- a/examples/02_hello_world.rs
+++ b/examples/02_hello_world.rs
@@ -1,7 +1,7 @@
 //! Basic hello world example.
 
 use ggez;
-use glam;
+use glam; // Requires feature "mint"
 
 use ggez::event;
 use ggez::graphics;

